main/git/APKBUILD

```diff
@@ -159,21 +153,26 @@ p4() {
 daemon() {
 	pkgdesc="Git protocol daemon"
 	depends="git=$pkgver-r$pkgrel"
-	replaces="git"
 
 	mkdir -p "$subpkgdir"/$_gitcoredir
 	mv "$pkgdir"/$_gitcoredir/git-daemon \
 		"$pkgdir"/$_gitcoredir/git-http-backend \
 		"$pkgdir"/$_gitcoredir/git-shell \
 		"$subpkgdir"/$_gitcoredir/
+}
+
+daemon_openrc() {
+	pkgdesc="Git protocol daemon"
+
+	default_openrc
```

Comment:
`rmdir "$pkgdr"/etc || :`

Reply:
Why is this rmdir needed?

Reply:
It's not strictly necessary. Removing the empty `etc` directory is just cleaner.